src/sflow_hooks.lua

```diff
@@ -123,7 +123,8 @@ typedef struct _SFSample {
     uint8_t eth_dst[8];
 
     /* vlan */
-    uint32_t in_vlan;
```

Comment:
Can you avoid changes with existing variables, please? That's fine to add new one because FFI can handle it but changing names can break backward compatibility.

Reply:
You are absolutely right. I returned backward compatibility.